packages/node/src/indexer/api.service.ts

```diff
@@ -360,12 +345,30 @@ export class ApiService implements OnApplicationShutdown {
   ): StorageKey[] {
     return calls.map((callMultiArg) => {
       if (callMultiArg instanceof Array) {
-        const [storageFunc, ...args] = callMultiArg;
-        const key = new StorageKey(api.registry, storageFunc.key(...args));
+        let key: StorageKey;
+        const storageFunc = callMultiArg[0];
```

Comment:
what's difference between this and `const [storageFunc, ...args] = callMultiArg;`, seems the same to me.